backend/codes/models.py

```diff
@@ -44,6 +44,12 @@ def pop_first():
 class Config(SingletonModel):
     force_stop = models.BooleanField(default=False)
     force_run = models.BooleanField(default=False)
+    udp_receive_run = models.BooleanField(default=False)
+    udp_key = models.PositiveIntegerField(default=99999, validators=[MinValueValidator(10000), MaxValueValidator(99999)])
```

Comment:
Lepiej, żeby nie miało wartości intowej - bardzo łatwo zgadnąć klucz. Aż za bardzo

Reply:
Lepiej zrobić to jako charfield i np. uuid4 wygenerować klucz

Reply:
Generalnie klucz jest tylko potrzebny w momencie kiedy serwer w ogóle jest uruchomiony, a jest on włączany z palca przez Admin panel.

1. Odpalasz serwer przez Admin panel i wpisujesz jaki ma być klucz
2. Ten sam klucz podajesz w app'ce podczas wysyłania aktualnej sceny

Więc jest on dodatkową wartością, na tyle prostą, żeby można było szybko przepisać do apki, ale to zmienię na ciąg znaków żeby trochę utrudnić, więc to jeszcze podmienię